src/kriscv/kdist/riscv-semantics/riscv.md

```diff
@@ -283,35 +283,33 @@ The following instructions behave analogously to their `I`-suffixed counterparts
 ```
 `BEQ` increments `PC` by `OFFSET` so long as the values in registers `RS1` and `RS2` are equal. Otherwise, `PC` is incremented by `4`.
 ```k
-  syntax Word ::= branchPC(oldPC: Word, offset: Int, branchCond: Bool) [function, total]
-  rule branchPC(PC,  OFFSET, COND) => PC +Word chop(OFFSET) requires COND
-  rule branchPC(PC, _OFFSET, COND) => PC +Word W(4)         requires notBool COND
+  syntax KItem ::= "#PC_BRANCH" "(" Int "," Bool ")"
+  rule <instrs> #PC_BRANCH(  OFFSET, COND) => .K ...</instrs>
+       <pc> PC => PC +Word chop(OFFSET) </pc>
+       requires COND
+  rule <instrs> #PC_BRANCH( _OFFSET, COND) => .K ...</instrs>
+       <pc> PC => PC +Word W(4) </pc>
+       requires notBool COND
```

Comment:
@jberthold, on the Booster backend, what's the difference between branching on rewrite rules vs branching on function rules?

Reply:
In particular, what happens in each case if `COND` is a symbolic term?

Reply:
Booster won't branch on function equations. For rewrite rules, it will branch on `COND`.